bundles/org.openhab.binding.fsinternetradio/src/main/java/org/openhab/binding/fsinternetradio/internal/radio/FrontierSiliconRadioConnection.java

```diff
@@ -52,7 +52,7 @@
     private String sessionId;
 
     /** http clients, store cookies, so it is kept in connection class. */
-    private HttpClient httpClient = null;
+    private HttpClient httpClient;
```

Comment:
```suggestion
    private final HttpClient httpClient;
```

Reply:
changed

Reply:
removed all @SuspressWarnings("null") - most of them were unnessesary, only a few now show a warning, but I think that's way better